x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/parameters_definition.ts

```diff
@@ -510,6 +510,9 @@ export const PARAMETERS_DEFINITION = {
           }),
         },
       ],
+      helpText: i18n.translate('xpack.idxMgmt.mappingsEditor.parameters.normalizerHelpText', {
+        defaultMessage: 'The name of the normalizer defined in your index settings.',
```

Comment:
Nit: because multiple normalizers could be defined in settings, I think we want this to read "a normalizer" instead of "the normalizer". And instead of "your index settings", maybe "the index's settings" makes more sense since we're still in the context of an index template and the user doesn't have an index yet?